demos/TicTacToe/Classes/TicTacToeScene.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <array>
```

Comment:
I don't think you need any of these in the header. None of them are needed for what is exposed in this header.